tests/CMakeLists.txt

```diff
@@ -248,6 +248,7 @@ add_examples(
   igraph_simplify
   igraph_topological_sorting
   igraph_transitivity
+  igraph_widest_paths
```

Comment:
You have added a unit test, not an example so please do not add it to the `add_examples(...)` block; those are meant for exercising code in `examples/simple`.

Reply:
Oops my bad! It's now in the correct spot (add_legacy_tests).